jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -256,21 +302,31 @@ private static void setSearchStrategy(String p) {
 
     private static PageLoadStrategy getPageLoadStrategy(String strategy) {
         switch (strategy.toLowerCase()) {
-            case "normal": return NORMAL;
-            case "none": return NONE;
-            case "eager": return EAGER;
+            case "normal":
+                return NORMAL;
+            case "none":
+                return NONE;
+            case "eager":
+                return EAGER;
         }
         return NORMAL;
     }
-    public static Properties getProperties(String path) {
-        // TODO use mergePath macos and windows
-        Properties pTest = PropertyReader.getProperties(mergePath(path));
-        Properties pTarget = PropertyReader.getProperties(mergePath("/../../target/classes/" + path));
-        if (pTarget.size() > 0)
-            return pTarget;
-        String propertiesPath = pTest.size() > 0
-                ? path
-                : "/../../target/classes/" + path;
-        return PropertyReader.getProperties(mergePath(propertiesPath));
+
+    public static Properties getCapabilities(String path) {
+        File propertyFile = new File(path);
+        Properties properties = new Properties();
+        if (propertyFile.exists()) {
+            try {
+                System.out.println("Property file found: " + propertyFile.getAbsolutePath());
+                properties.load(new FileInputStream(propertyFile));
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        } else {
+            // TODO use mergePath macos and windows
+            String propertyFilePath = "/../../target/classes/" + path;
+            properties = PropertyReader.getProperties(propertyFilePath);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'PropertyReader.getProperties' due to existing static import 'com.epam.jdi.tools.PropertyReader.getProperties'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4665121)